scripts/install

```diff
@@ -32,6 +32,7 @@ report_stats="False"
 #=================================================
 
 domain=$YNH_APP_ARG_DOMAIN
+server_name=$YNH_APP_ARG_SERVER_NAME
```

Comment:
I think you need to save this settings and use it in the upgrade and config scripts.